File: 프로그래머스/lv1/42748. K번째수/K번째수.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> solution(vector<int> array, vector<vector<int>> commands) {
    vector<int> answer;
    for(auto com : commands) {
        int i = com[0];
        int j = com[1];
        int k = com[2];
        vector<int> v;
        //cout<<i<<" "<<j;
        int length = j-i+1;
        //cout<<length;
        while(length--) {
            v.push_back(array[i-1]);
            cout<<array[i-1]<<" ";
            i++;
        }
        cout<<'\n';
        sort(v.begin(),v.end());
        answer.push_back(v[k-1]);
    }
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    var answer = [];
    for(c of commands) {
        let arr = array.slice(c[0]-1, c[1]);
        console.log(arr);
        arr.sort((x,y)=>x-y);
        answer.push(arr[c[2]-1]);
    }
    return answer;
}